README.md

```diff
@@ -53,6 +53,9 @@ If you find a problem or missing feature, open an issue or a pull requests on ht
 
 To have a higher chance of issue being solved, please attach a log file. To record one, go to `settings wheel -> System settings -> Logging -> Enable Debug Logging` and follow the procedure at http://kodi.wiki/view/Log_file/Easy
 
-## Note for Raspberry Pi users:
+# Note for ARM users #
+## Nvidia Shield / most Android boxes ##
+- _Ambilight mode_ doesn't properly work with some HD codecs (usually x265 >1080p) when "allow hardware acceleration - Mediacodec (Surface)" is enabled.
+## Raspberry Pi's: ##
  - Save the add-on configuration by exiting Kodi before shutting down the Pi completely
- - Ambilight mode doesn't work on a Raspberry Pi due to the way it renders video
+ - _Ambilight mode_ doesn't work on a _Raspberry Pi_ due to the way it renders video
```

Comment:
Newer codecs don't work because Kodi doesn't even get the image - it's all hardware decoded. I guess it might be possible to find codecs that don't work and notify the user that for the particular movie ambilight will be disabled and just don't trigger the group at all.